<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectArgumentsWidget</class>
 <widget class="QWidget" name="SelectArgumentsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>586</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="1">
    <widget class="SelectVariableWidget" name="widget_selectVariable" native="true"/>
   </item>
   <item row="0" column="0">
    <widget class="SelectVariableGroupWidget" name="widget_selectVariableGroup" native="true"/>
   </item>
   <item row="1" column="0" colspan="2">
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="2">
      <widget class="QPushButton" name="pushButton_confirm">
       <property name="text">
        <string>确认</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="1">
      <widget class="QPushButton" name="pushButton_cancel">
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>SelectVariableGroupWidget</class>
   <extends>QWidget</extends>
   <header>SelectVariableGroupWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>SelectVariableWidget</class>
   <extends>QWidget</extends>
   <header>SelectVariableWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
